python/taichi/ui/gui.py

```diff
@@ -13,6 +13,16 @@ class Gui:
     def __init__(self, gui) -> None:
         self.gui = gui  #reference to a PyGui
 
+    def __enter__(self, name, x, y, width, height):
```

Comment:
`__enter__` does not accept other arguments than `self`
https://docs.python.org/zh-cn/3/reference/datamodel.html#object.__enter__

Reply:
Oops, I will revisit the docs of `__enter__`.

Reply:
@frostming Fixed, PTAL